src/validation/ndrop/update_from_explmix.cpp

```diff
@@ -194,36 +164,57 @@ void update_from_explmix(Ensemble *ensemble) {
       }
     }
 
-    // FIXME: done?
-    // TODO: need thread team here
-    ndrop::update_from_explmix(team, dtmicro, csbot, cldn, zn, zs, ekd, nact,
-                               mact, qcld, raercol, raercol_cw, nsav, nnew,
-                               nspec_amode, mam_idx, overlapp, overlapm, ekkp,
-                               ekkm, qncld, srcn, source);
-
-    // TODO: ColumnView-ify the output sequence
+    auto team_policy = ThreadTeamPolicy(1u, Kokkos::AUTO);
```

Comment:
Can you double-check this team policy? The second argument is the team size, and if `update_from_explmix` doesn't have any parallel dispatches inside it, this second argument should be 1.

Reply:
`update_from_explmix` has several parallel_fors like this in it:
```
  Kokkos::parallel_reduce(
      Kokkos::TeamThreadRange(team, pver - top_lev + 1),
      [&](int kk, Real &min_val) {
```
https://github.com/eagles-project/mam4xx/blob/main/src/mam4xx/ndrop.hpp#L1217 
which do involve kp1 and km1 levels so do we want to keep it as `Kokkos::AUTO`?

Reply:
At the very least, let's change it to nlev (or its equivalent).

Reply:
We have nested parallelism in this update_from_explmix. Thus, we need to add `team_barriers`. Note this test passes the test in both cpu and gpu.  `Kokkos::AUTO` set two parameters` team size` and `vector size`. 

In addition, 

Note that this code, kk starts in `0 `and ends `pver - top_lev + 1`; However, inside of the parrallel_for, we use k as `const int k = top_lev - 1 + kk;`

```
Kokkos::parallel_reduce(
      Kokkos::TeamThreadRange(team, pver - top_lev + 1),
      [&](int kk, Real &min_val) {
        const int k = top_lev - 1 + kk;
```

Thus, k starts top_lev - 1 and ends in pver-1

Reply:
Okay, so good to merge as is?

Reply:
Merge away!